src/view/container/device/Device.tsx

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as React from "react";
+import Simulator from "../../components/Simulator";
+import {
+    TOOLBAR_ICON_ID
+} from "../../components/toolbar/SensorModalUtils";
+import ToolBar from "../../components/toolbar/ToolBar";
+import * as TOOLBAR_SVG from "../../svgs/toolbar_svg";
+
+const CPX_TOOLBAR_BUTTONS: Array<{ label: any, image: any }> = [{
+    image: TOOLBAR_SVG.SLIDER_SWITCH_SVG,
+    label: TOOLBAR_ICON_ID.SWITCH,
+}, {
+    image: TOOLBAR_SVG.PUSH_BUTTON_SVG,
+    label: TOOLBAR_ICON_ID.PUSH_BUTTON,
+},
+{
+    image: TOOLBAR_SVG.RED_LED_SVG,
+    label: TOOLBAR_ICON_ID.RED_LED,
+}, {
+    image: TOOLBAR_SVG.SOUND_SVG,
+    label: TOOLBAR_ICON_ID.SOUND,
+}
+    , {
+        image: TOOLBAR_SVG.TEMPERATURE_SVG,
+    label: TOOLBAR_ICON_ID.TEMPERATURE,
+}, {
+    image: TOOLBAR_SVG.LIGHT_SVG,
+    label: TOOLBAR_ICON_ID.LIGHT,
+}
+    , {
```

Comment:
nit: these brackets are all messed up and inconsistent

Reply:
thanks for the heads up. I'll actually make it a task to have proper linting rules and set up so it doesnt slow us down in the future